### ISSUE TITLE
ansible apt module and dpkg

### ISSUE THREAD

**Issue by dolanjs** [freedomofpress/securedrop#848]
When using the ansible apt module to install local deb packages and to resolve those local package dependencies keep getting an error when using variables with lists. The error and associated tasks are listed in commit 79d369bd18d8c7c77fd74aee666bb948c7574035. A Temp work around was implemented in commit  b31019c98e32fec6528e44b052bf18e29b688208 to use the ansible shell module instead.

**Comment by conorsch** [comment]
This is indeed a bug, and should be resolved when Ansible 2 ships, but the standard workaround is not to use `shell`. You can instead use `with_indexed` to prevent the flattening on the list that the `apt` module enforces by default, like so:

```
 - name: if defined install local deb packages
-  shell: "dpkg -i /root/{{ item }} || true && apt-get -f install -y"
-  with_items: local_deb_packages
+  apt:
+    deb: /root/{{ item.1 }}
+  with_indexed_items: local_deb_packages
   tags:
     - install_local_pkgs
```

Hacky, but better than `shell`.

**Comment by conorsch** [comment]
Resolved via #1196, specifically 5ee6ff1b9b7961d7a9265c81d0eac40d4a253389.
